embedded-minio/src/test/java/com/playtika/test/minio/EmbeddedMinioBootstrapConfigurationTest.java

```diff
@@ -65,7 +65,7 @@ public void shouldEmulateLatency() throws Exception {
     }
 
     private void writeFileToMinio(String fileName, String path) throws Exception {
-        minioClient.putObject(BUCKET, fileName, path, null, null, null, null);
+        minioClient.putObject(BUCKET, fileName, path, null);
```

Comment:
`PutObjectOptions` should be passed.

Reply:
ok, but it fails on
`
 @Before
    public void setUp() throws Exception {
        if (!minioClient.bucketExists(BUCKET)) {
            minioClient.makeBucket(BUCKET);
        }
    }
`